### PR TITLE
RA-1461: Removed History_Of functionality and added labels

### PR DESCRIPTION
**Pull request by haripriya999**
# Removed History_Of functionality

## Description of what I changed
1.  Removed the History_of functionality

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1461

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp [hunk 1]
```diff
@@ -46,11 +46,13 @@
     <div class="horizontal">
         <ul id="concept-and-date">
             <li class="group">
+                <h3 class="inline">${ui.message('coreapps.conditionui.condition')} </h3>
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Is this addition part of removing history of?

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp [hunk 2]
```diff
@@ -46,11 +46,13 @@
...
                 <coded-or-free-text-answer id="conceptId" class="concept"
                                            concept-classes="8d4918b0-c2cc-11de-8d13-0010c6dffd0f,8d492954-c2cc-11de-8d13-0010c6dffd0f,8d492b2a-c2cc-11de-8d13-0010c6dffd0f,8d491a9a-c2cc-11de-8d13-0010c6dffd0f"
                                            ng-model="concept"/>
             </li>
             <li class="group">
+                <h3 class="inline">${ui.message('coreapps.conditionui.onsetdate')} </h3>
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Is this addition part of removing history of?

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp (hunk 1 above)
Comment (haripriya999) [reply omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: No, This is to address the issue of labeling the fields. As it was a very minor change I included it in this PR.

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp (hunk 2 above)
Comment (haripriya999) [reply omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: So is the case with this one

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp (hunk 1 above)
Comment (dkayiwa) [reply omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Then that should be a separate ticket and hence pull request.

**Comment by dkayiwa** [comment]
Can you also assign the ticket to yourself?

**Comment by haripriya999** [comment]
I claimed the ticket :)

**Comment by dkayiwa** [comment]
Can you also address the rest of my comments?

**Comment by haripriya999** [comment]
Sorry, I missed that comment above. I updated the PR.

**Comment by dkayiwa** [comment]
On the JIRA ticket, attach screenshots showing how the interface was before and after your changes.

**Comment by haripriya999** [comment]
Added screenshots to JIRA :)

**Pull request merged**